.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: python
+python:
+  - "2.7"
+  - "3.6"
+services:
+  - docker
+before_install:
+  - pushd ..
+  - git clone https://github.com/IBM/MAX-Question-Answering-Web-App.git
+  - cd MAX-Question-Answering-Web-App
+  - docker build -t max-question-answering-web-app .
+  - docker run -it -d -p 5000:5000 max-question-answering-web-app
+  - popd
+  - sleep 30
+  - pip install -r requirements.txt
+install:
+  - python app.py &
+  - sleep 90
+before_script:
+  - pip install -r test-requirements.txt
```

Comment:
This should be in the `install` section

Reply:
Let's start with basic testing first (ignoring Docker tests) and make sure that flake8 doesn't report any issues.